<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGAL 3D Combinatorial Map</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <layout class="QHBoxLayout"/>
    </item>
    <item>
     <widget class="Viewer" name="viewer" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImportOFF"/>
    <addaction name="actionAddOFF"/>
    <addaction name="separator"/>
    <addaction name="actionImport3DTDS"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>Operations</string>
    </property>
    <addaction name="actionSubdivide"/>
    <addaction name="actionCreateCube"/>
    <addaction name="actionDisplayInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperations"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportOFF">
   <property name="text">
    <string>Import OFF</string>
   </property>
  </action>
  <action name="actionAddOFF">
   <property name="text">
    <string>Add OFF</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSubdivide">
   <property name="text">
    <string>Subdivide</string>
   </property>
  </action>
  <action name="actionCreateCube">
   <property name="text">
    <string>Create cube</string>
   </property>
  </action>
  <action name="actionImport3DTDS">
   <property name="text">
    <string>Import 3DTS</string>
   </property>
  </action>
  <action name="actionDisplayInfo">
   <property name="text">
    <string>Display info</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header>Viewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Combinatorial_map_3.qrc"/>
 </resources>
 <connections/>
</ui>
